helix-term/src/commands.rs

```diff
@@ -287,6 +287,7 @@ impl MappableCommand {
         select_mode, "Enter selection extend mode",
         exit_select_mode, "Exit selection mode",
         goto_definition, "Goto definition",
+        goto_definition_other_window, "Goto definition other window",
```

Comment:
This could be clearer, e.g. "Goto definition in the next pane"

Reply:
Thanks! Just pushed up that change.